app/jobs/address_proofing_job.rb

```diff
@@ -1,11 +1,17 @@
 class AddressProofingJob < ApplicationJob
   include JobHelpers::FaradayHelper
+  include JobHelpers::StaleJobHelper
 
   queue_as :default
 
   def perform(user_id:, issuer:, result_id:, encrypted_arguments:, trace_id:)
     timer = JobHelpers::Timer.new
 
+    if stale_job?(enqueued_at)
+      notify_stale_job
```

Comment:
I think we should raise an exception so it triggers the failed worker alarm

Reply:
ah not just notify directly? will do

Reply:
Updated in e6f1a95f455317ee0ca0b8f5aea09a35e0d44f15